### ISSUE TITLE
Width 100% in Select component not working properly inside InputGroup

### ISSUE THREAD

**Issue by mauriciosoares** [ant-design/ant-design#7662]
### Version
2.13.2

### Environment
macOS Sierra 10.12.6 - Chrome 60.0.3112.113

### Reproduction link
[https://codepen.io/anon/pen/ZXQgYo?editors=0010](https://codepen.io/anon/pen/ZXQgYo?editors=0010)

### Steps to reproduce
1 - Select the second option for both `<Select />` components

### What is expected?
Both components should have the same width, since their parent contains width 200px, and the `<Select />` components have 100%

### What is actually happening?
The `<Select />` component inside `<InputGroup />` ignores the 100% width when the content inside the option is too big. It ends up with the same width of the content

**Comment by afc163** [comment]
```jsx
          <InputGroup compact>
            <Select>
              {options}
            </Select>
          </InputGroup>
```

That is a weird usage, please provide a real world example.

**Comment by mauriciosoares** [comment]
@afc163 In my UI, I have some buttons beside that Select that acts according to the value of the Select. That's why I grouped them. Check the screenshot below:

![screen shot 2017-09-19 at 16 49 23](https://user-images.githubusercontent.com/2321259/30598631-8fcd751a-9d5a-11e7-8c8f-f4ba3c9a8380.png)

That's what happens when I select an item that's too big.
![screen shot 2017-09-19 at 16 49 32](https://user-images.githubusercontent.com/2321259/30598632-8fee972c-9d5a-11e7-80f6-6d23b1ecdc1b.png)

I updated the CodePen. The idea for using InputGroup is that it makes things prettier in that case. And since in the examples page, [there are examples](https://ant.design/components/input/#components-input-demo-group) using the Select component, I considered that a bug
